### PR TITLE
Fix apt-key deprecation warning on bullseye

### PR DESCRIPTION
**Pull request by ecdye**
Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

**Comment by ecdye** [comment]
@mstormi Wait to merge until the issue with GitHub Actions tests is resolved so we can run them (GitHub is currently experiencing issues with actions right now).

File: functions/helpers.bash [hunk 1]
```diff
@@ -42,18 +42,15 @@ cond_echo() {
...
 ##
 add_keys() {
-  local repoKey
-
-  repoKey="$(mktemp "${TMPDIR:-/tmp}"/openhabian.XXXXX)"
+  local repoKey="/usr/share/keyrings/${2}.gpg"
 
   echo -n "$(timestamp) [openHABian] Adding required keys to apt... "
-  cond_redirect wget -qO "$repoKey" "$1"
-  if cond_redirect apt-key add "$repoKey"; then
```
Comment (mstormi) [inline functions/helpers.bash:54]: deleted L54 but no replacement ?

**Review by mstormi — CHANGES_REQUESTED** [review]

File: functions/helpers.bash (hunk 1 above)
Comment (ecdye) [reply functions/helpers.bash:54]: Its a one liner on line 52 now as apt-key has been deprecated in bullseye.

**Review by mstormi — CHANGES_REQUESTED** [review]

**Comment by mstormi** [comment]
(interesting:
I had commented some hours ago the comment was marked as Pending and didn't get to you until I re-requested a review)

File: functions/helpers.bash (hunk 1 above)
Comment (ecdye) [reply functions/helpers.bash:54]: Yes, it works on both. It simply changes the way apt checks the signature.

**Review by mstormi — APPROVED** [review]
LGTM then but DNT

**Comment by mstormi** [comment]
is that Github actions thingy resolved ?

**Comment by ecdye** [comment]
> is that Github actions thingy resolved ?

Probably, I'm going to force push it to make GH Actions run again just to be sure

**Review by mstormi — APPROVED** [review]
LGTM DNT

**Pull request merged**